src/listeners.ts

```diff
@@ -3,17 +3,18 @@ import Webhooks from "@octokit/webhooks";
 import { lintCommitMessage } from "./lint";
 import { Commit as GithubCommit } from "types/github";
 import { CommitInfo, saveCommit } from "./db";
-import Logger, { addInvocationBreadcrumb } from "./logger";
+import { addInvocationBreadcrumb } from "./logger";
 
 async function processCommit(
   githubCommit: GithubCommit,
   repoName: string
 ): Promise<CommitInfo> {
-  Logger.captureMessage("Receive new commit", (scope) =>
-    scope.setExtra("data", {
-      repoName,
-      hash: githubCommit.id,
-    })
+  console.log(
```

Comment:
For logging, we should be using structured logging. You can refer to https://probot.github.io/docs/logging/ for details.